pkg/bootstrap/docker/up.go

```diff
@@ -857,7 +869,7 @@ func (c *ClientStartConfig) InstallLogging(out io.Writer) error {
 	if len(publicMaster) == 0 {
 		publicMaster = c.ServerIP
 	}
-	return c.OpenShiftHelper().InstallLogging(f, publicMaster, openshift.LoggingHost(c.RoutingSuffix, c.ServerIP), c.Image, c.ImageVersion)
+	return c.OpenShiftHelper().InstallLogging(f, publicMaster, openshift.LoggingHost(c.RoutingSuffix, c.ServerIP), defaultLoggingImagePrefix, c.ImageVersion)
```

Comment:
Due to a bug the prefix for logging and metrics != the prefix for OpenShift components in RHEL builds https://bugzilla.redhat.com/show_bug.cgi?id=1416240